### PR TITLE
Testing/player list

### PR DESCRIPTION
**Pull request by tysnj**
## Is this a fix or a feature?
Both

## What is the change?
Add playerList tests

## What does it fix?
PlayerList also had a bug where it wasn't showing the proper list of players in community views.

## Where should the reviewer start?
checkout `testing/playerList` branch

## How should this be tested?
Run cypress tests for the playerList test suite

### CODE REVIEW

File: src/APICalls/APICalls.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const getAllEvents = (playerId) => {
 }
 
 export const postEvent = (courseId, date, teeTime, openSpots, numHoles, isPrivate, hostId, selectedFriends) => {
-  return fetch('http://3d8bf4156a8c.ngrok.io/api/v1/events', {
+  return fetch('http://861341e035fa.ngrok.io/api/v1/event', {
```
Comment (jon-schlandt) [inline src/APICalls/APICalls.js:48]: Nice catch fixing this - might want to utilize the denEnv variable to make it easier to change once the ngrok server inevitably changes (but I think Eric is handling this on his end).

**Review by jon-schlandt — APPROVED** [review]
Nice work getting the friend feature working! 

One thing I might suggest we look into is having some sort of message like 'You don't have any friends, go make some!' or something when a user does not have any added friends.

File: cypress/integration/playerList.spec.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+describe('Player List', () => {
+  beforeEach('setup stubs and visit Dashboard', () => {
+    cy.viewport(1920, 1080)
```
Comment (errabun) [inline cypress/integration/playerList.spec.js:3]: Didn't know you could change the viewport size on cypress...this is awesome

File: src/components/App/App.js [hunk 3]
```diff
@@ -113,6 +99,9 @@ function App() {
           path='/event-form'
           render={() => <EventForm courses={courses} friends={friends} hostId={hostPlayer.id} />}
         />
+        <Route exact path='/'>
+          <Redirect to='/dashboard' /> // This is a quick fix, might want to default web server to http://localhost:3000/dashboard if possible
```
Comment (errabun) [inline src/components/App/App.js:103]: Lets remember to delete this comment

**Review by errabun — APPROVED** [review]

**Pull request merged**